### ISSUE TITLE
iOS / flutter 1.20 / cannot build / Share Extension/Info.plist to be fixed ?

### ISSUE THREAD

**Issue by monsieurtanuki** [KasemJaffer/receive_sharing_intent#77]
If I use the following suggested values in `Share Extension/Info.plist`:
```
  <key>CFBundleShortVersionString</key>
  <string>$(FLUTTER_BUILD_NAME)</string>
  <key>CFBundleVersion</key>
  <string>$(FLUTTER_BUILD_NUMBER)</string>
```
I get the following error when I run my app:
````
Launching lib/main.dart on iPhone 8 Plus in debug mode...
Running pod install...
Running Xcode build...
Xcode build done.                                           88,0s
Unable to install /Users/myself/AndroidStudioProjects/flutter_app/build/ios/iphonesimulator/Runner.app on F037BDC6-54C2-4B52-931B-588F5DD62572. This is sometimes caused by a malformed plist file:
ProcessException: Process exited abnormally:
An error was encountered processing the command (domain=IXErrorDomain, code=2):
Failed to create plugin placeholder for /Users/myself/AndroidStudioProjects/flutter_app/build/ios/iphonesimulator/Runner.app/PlugIns/Share Extension.appex
Failed to create promise.
Underlying error (domain=IXErrorDomain, code=2):
	Failed to set placeholder attributes com.example.flutterApp.Share-Extension
	Failed to create promise.
  Command: /usr/bin/xcrun simctl install F037BDC6-54C2-4B52-931B-588F5DD62572 /Users/myself/AndroidStudioProjects/flutter_app/build/ios/iphonesimulator/Runner.app
Error launching application on iPhone 8 Plus.
````

But the app gets built and runs with the following values:
````
  <key>CFBundleShortVersionString</key>
  <string>1</string>
  <key>CFBundleVersion</key>
  <string>1.0</string>
````

**Comment by naoki0719** [comment]
Open Xcode and select BuildPhase > Embed App Extensions.

Check "Copy only when installing".

flutter clean
flutter run

This worked for me.

**Comment by monsieurtanuki** [comment]
@naoki0719 Thank you! To be more precise, it looks like it's "Targets / Runner / Build Phases / Embed App Extensions" : check "Copy only when installing".
It seemed to work on one project, I mean, at least, the build was successful.
Then I tried on another project, and I got trapped in a concurrency nonsense race between Android Studio and Xcode on file `Runner.xcodeproj`. But that's another story.
